core/string/translation.h

```diff
@@ -104,6 +114,33 @@ class TranslationServer : public Object {
 	StringName translate(const StringName &p_message, const StringName &p_context = "") const;
 	StringName translate_plural(const StringName &p_message, const StringName &p_message_plural, int p_n, const StringName &p_context = "") const;
 
+	StringName pseudolocalize(const StringName &p_message) const;
+	String get_override_string(String &message) const;
+	String double_vowels(String &message) const;
+	String replace_with_accented_string(String &message) const;
+	String wrap_with_fakebidi_characters(String &message) const;
+	String add_padding(String &message, int length) const;
+	const wchar_t *get_accented_version(char c) const;
```

Comment:
Shouldn't this be using `CharType` instead of `wchat_t` and `char`? @bruvzg

Reply:
It will work with `wchar_t` (there's nothing outside its range used here), but it causes extra  UTF-16 → UTF-32 conversion every time it's added to string, using `char32_t` and `U"x"` literals is better.